ntbs-integration-tests/Helpers/HttpClientExtensions.cs

```diff
@@ -34,6 +34,25 @@ public static class HttpClientExtensions
             return client.SendAsync(submission);
         }
 
+        public static Task<HttpResponseMessage> SendGetAsync(

```

Comment:
Get forms don't use an authentication token by default and so don't need the extra logic of supplying one as found int SendAsync(). Would it be preferable to have separate methods for get forms/forms without an authentication token or to have a flag as a parameter to the method to tell it whether or not an auth token is required?